etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/HappyPathMockClass.groovy

```diff
@@ -5,7 +5,6 @@ import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhir
 import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper
 import org.hl7.fhir.r4.model.Bundle
 import org.hl7.fhir.r4.model.Coding
-import org.hl7.fhir.r4.model.MessageHeader
 
 class HappyPathMockClass implements CustomFhirTransformation {
```

Comment:
I know this class wasn't exactly part of this PR, but `HappyPathMockClass` doesn't exactly tell me what it does.

Reply:
@saquino0827 I think you added this one?

Reply:
Ah yes, I can rename it. It is a Happy Path class for ensuring the TransformationEngine runs a custom class when it is given a valid class as input.

Reply:
This has been renamed